src/Interpolation.elm

```diff
@@ -286,6 +293,40 @@ hslImpl hueInt from to =
     map4 Color.hsla (hueInt start.hue end.hue) (float start.saturation end.saturation) (float start.lightness end.lightness) (float start.alpha end.alpha)
 
 
+{-| Interpolates between two Color values using the [CIELAB color space](https://en.wikipedia.org/wiki/CIELAB_color_space) color space.
+-}
+lab : Color -> Color -> Interpolator Color
+lab from to =
+    let
+        start =
+            toLab from
+
+        end =
+            toLab to
+    in
+    map4 fromLabPositional (float start.l end.l) (float start.a end.a) (float start.b end.b) (float start.alpha end.alpha)
+
+
+{-| Interpolates between two Color values using the [CIE Lch(ab)](https://en.wikipedia.org/wiki/HCL_color_space) color space.
+-}
+hcl : Color -> Color -> Interpolator Color
+hcl from to =
+    let
+        start =
+            toHcl from
+                |> Debug.log "start -- this seems OK"
+
+        end =
+            toHcl to
+                |> Debug.log "end -- this seems OK"
+    in
+    map4 fromHclPositional
+        (float start.hue end.hue)
```

Comment:
Doesn't this one need to use the `hue` function?

```suggestion
        (hue start.hue end.hue)
```

See [here](https://github.com/d3/d3-interpolate/blob/master/src/hcl.js#L6) for reference.